src/Core/Search/Discussion/Gambit/CreatedGambit.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Flarum\Core\Search\Discussion\Gambit;
+
+use Flarum\Core\Search\AbstractRegexGambit;
+use Flarum\Core\Search\AbstractSearch;
+use Flarum\Core\Search\Discussion\DiscussionSearch;
+use LogicException;
+
+class CreatedGambit extends AbstractRegexGambit
+{
+    /**
+     * http://stackoverflow.com/a/8270148/3158312
+     *
+     * @FIXME: Find better regular expression
```

Comment:
It passess for values like 25th hour or 70th minute. Regexes are not my good side so I only tested a few regexes and this will work fine too, it will just generate query which returns nothing (in case like 25th hour), but I think that there should be more strict pattern.